stores/sql_test.go

```diff
@@ -110,3 +112,47 @@ func TestConsensusReset(t *testing.T) {
 		t.Fatal("wrong id", db.chainIndex.ID, types.BlockID{})
 	}
 }
+
+type queryPlanExplain struct {
+	ID      int    `json:"id"`
+	Parent  int    `json:"parent"`
+	NotUsed bool   `json:"notused"`
+	Detail  string `json:"detail"`
+}
+
+func TestQueryPlan(t *testing.T) {
```

Comment:
I'm planning to F/U with a migrations PR. I think we have to test both clean installs and migrations of an existing install from now on. Doesn't have to be this PR per se I think, but definitely we need more rigorous testing of our migrations going forward...

I think this is a pretty good little unit test to copy paste all over the place whenever we introduce important queries. Having assertions on the execution plan is something we should've done sooner.